UnixBench/USAGE

```diff
@@ -65,6 +65,11 @@ so that randomly-churning background processes don't randomise the results
 too much.  This is particularly true for the graphics tests.
 
 
+If you need to change the directories, use "Environment variables" about below.
```

Comment:
Please change the usage text to read:

Output can be specified by using the following variables:

  * "UB_RESULTDIR"        : Absolute path of output directory of result files.
  * "UB_TMPDIR"               : Absolute path of temporary files for IO tests.
  * "UB_OUTPUT_FILE_NAME"  : Output file name. If exists it will be overwritten.